src/app.js

```diff
@@ -30,53 +30,54 @@ client.on('ready', () => {
 });
 
 // Create an event listener for messages
-client.on('message', msg => {
-  const { content, author, channel, member } = msg;
+client.on('message', async msg => {
+  // Ignore messages from the bot
+  if (msg.author.bot) return;
 
-  // Returns true if the message content matches any of
-  // the given commands
-  const commandMatches = commands => {
-    if (Array.isArray(commands)) {
-      for (let i = 0; i < commands.length; i++) {
-        if (`${prefix}${commands[i]}` === content.split(' ')[0]) return true;
-      }
-      return false;
-    }
+  //Ignore any message that does not start with prefix
+  if (msg.content.indexOf(prefix) !== 0) return;
```

Comment:
Although this works, `msg.content` has a `startsWith` function that is more readable:

```js
if (content.startsWith(prefix)) return
```